レビュー用PR

kiy0taka opened the pull request.

kentanakayrglm committed:
正式ブランチへの最新版コミット

chihiro-adachi committed:
Merge pull request #1 from KenTanaka/main

正式ブランチへの最新版コミット

kentanakayrglm committed:
資料更新

kentanakayrglm committed:
・1207 レビュー項目取込み（認証電話番号をユニークとする対応）
・ログイン後の二段階認証完了後、流入元URLへリダイレクトさせる対応
・全体的のリファクタリング

kentanakayrglm committed:
・リファクタリング反映（分離前）
・重要ルーティング/除外ルーティング設定可能版
・EC-CUBEペイメント導入によるテスト完了版

kentanakayrglm committed:
ログイン・重要操作時の具体的認証（SMS/AP）認証は子プラグインで実装

chihiro-adachi committed:
Merge pull request #2 from KenTanaka/develop/refactor1206

レビュー項目取込み・リファクタリング

kentanakayrglm committed:
・不要ロジックの削除
・Cookie有効期間の見直し

KenTanaka committed:
Merge pull request #2 from KenTanaka/develop/refactor1206

Develop/refactor1206

kiy0taka commented on Controller/Admin/ConfigController.php:
```diff
@@ -0,0 +1,88 @@
+use Eccube\Controller\AbstractController;
+use Plugin\TwoFactorAuthCustomer42\Form\Type\TwoFactorAuthConfigType;
+use Plugin\TwoFactorAuthCustomer42\Repository\TwoFactorAuthConfigRepository;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
```
全体的に `@Route` は `Symfony\Component\Routing\Annotation\Route` の方を利用してください

KenTanaka replied:
修正しました。

kiy0taka commented on Controller/Admin/ConfigController.php:
```diff
@@ -0,0 +1,88 @@
+    /**
+     * @var BaseInfoRepository
+     */
+    protected $baseInfoRepository;
```
全体的に利用していないプロパティやローカル変数があるので削除お願いします。

KenTanaka replied:
見直しました。

kiy0taka commented on Controller/TwoFactorAuthCustomerController.php:
```diff
@@ -0,0 +1,312 @@
+use Eccube\Controller\AbstractController;
+use Eccube\Entity\Customer;
+use Eccube\Repository\CustomerRepository;
+use Plugin\TwoFactorAuthCustomer42\Form\Type\TwoFactorAuthConfigType;
```
利用していないuseの削除もお願いします。

KenTanaka replied:
不要分は削除しました。

kiy0taka commented on Controller/TwoFactorAuthCustomerController.php:
```diff
@@ -0,0 +1,312 @@
+                    log_warning('[デバイス認証(SMS)] 既に認証済みの電話番号指定');
+                }
+                $response = new RedirectResponse(
+                    $this->generateUrl(
```
`return $this->redirectToRoute(...)` でお願いします。

KenTanaka replied:
修正しました。

kiy0taka commented on Controller/TwoFactorAuthCustomerController.php:
```diff
@@ -0,0 +1,312 @@
+                    $phoneNumber = $this->session->get(CustomerTwoFactorAuthService::SESSION_AUTHED_PHONE_NUMBER);
+                    // ワンタイムトークン一致
+                    // デバイス認証完了
+                    $Customer->setDeviceAuthed(true);
```
ここでも電話番号が認証済みになっていないかの確認が必要かと思います。

sai-gillingham replied:
こちらの関数を
Controller/CustomerPersonalValidationController.phpクラスに移動しましたが、
電話番号が認証済みになっていないかという条件分岐を追加しました。

kiy0taka replied:
`findOneBy`ではなく`count`を利用してください。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/blob/9e48ad2dbb15ec9ccb2cf3a7dffbe46427f0fba8/Controller/CustomerPersonalValidationController.php#L107

sai-gillingham replied:
修正しました。

kiy0taka commented on Controller/TwoFactorAuthCustomerController.php:
```diff
@@ -0,0 +1,312 @@
+     * 
+     * @return boolean
+     */
+    protected function isAuth() 
```
二要素認証済みかどうかを判定するメソッドだと思うので、`isTwoFactorAuthed` というメソッド名がわかりやすいかと思います。

sai-gillingham replied:
修正しました。

kiy0taka commented on Entity/BaseInfoTrait.php:
```diff
@@ -0,0 +1,57 @@
+     * @var bool
+     * @ORM\Column(name="option_activate_sms", type="boolean", nullable=false, options={"default":false})
+     */
+    private bool $option_activate_sms;
```
デバイス認証という言葉で統一するために `option_activate_device` または `option_device_acitevate` が良いかと思います。

sai-gillingham replied:
関数名を`option_activate_device`に更新しました。

kiy0taka commented on Entity/CustomerTrait.php:
```diff
@@ -0,0 +1,197 @@
+     *
+     * @ORM\Column(name="two_factor_auth", type="boolean", nullable=false, options={"default":false})
+     */
+    private bool $two_factor_auth = false;
```
`two_factor_auth_use` または `two_factor_enabled` が良いかと思います。

sai-gillingham replied:
two_factor_authは不要になったため、
two_factor_authがなくなりました。

kiy0taka commented on Entity/CustomerTrait.php:
```diff
@@ -0,0 +1,197 @@
+        // TODO: なんちゃって
+        $token = '';
+        for ($i = 0; $i < 6; $i++) {
+            $token .= (string)rand(0, 9);
```
[random_int](https://www.php.net/manual/ja/function.random-int.php) でお願いします。

sai-gillingham replied:
修正しました。

kiy0taka replied:
トークン長を変更可能にするのであれば、services.ymlを使うようにお願いします。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/blob/9e48ad2dbb15ec9ccb2cf3a7dffbe46427f0fba8/Service/CustomerTwoFactorAuthService.php#L405-L412

kiy0taka commented on Entity/TwoFactorAuthConfig.php:
```diff
@@ -0,0 +1,274 @@
+     *
+     * @ORM\Column(name="include_route", type="text", nullable=true)
+     */
+    private $include_route = null;
```
複数形 `include_routes` がいいかと思います。

sai-gillingham replied:
修正しました。

kiy0taka commented on Entity/TwoFactorAuthConfig.php:
```diff
@@ -0,0 +1,274 @@
+     *
+     * @ORM\Column(name="exclude_route", type="text", nullable=true)
+     */
+    private $exclude_route = null;
```
こちらも複数形になるかと思います。

sai-gillingham replied:
除外ルーティングがなくなったため、
こちらのコードもなくなりました。

kiy0taka commented on Entity/TwoFactorAuthConfig.php:
```diff
@@ -0,0 +1,274 @@
+        $routes = $this->getRoutes($this->getIncludeRoute());
+
+        if (in_array($route, $routes)) {
+            $routes = array_splice(array_search($route, $routes, true));
```
`array_splice`の引数は以下が正しいかと思いますが、`array_search` がfalseの場合も考慮する必要があるかと思います。
```
array_splice($routes, array_search($route, $routes, true), 1)
```

sai-gillingham replied:
`array_splice`ではなく
`array_diff`を利用するようになりました。

kiy0taka commented on Entity/TwoFactorAuthConfig.php:
```diff
@@ -0,0 +1,274 @@
+        $routes = $this->getRoutes($this->getExcludeRoute());
+
+        if (in_array($route, $routes)) {
+            $routes = array_splice(array_search($route, $routes, true));
```
こちらも同様かと思います。

sai-gillingham replied:
除外ルーティングがなくなったため、
こちらの関数の中身がなくなりました。

kiy0taka commented on Entity/TwoFactorAuthType.php:
```diff
@@ -0,0 +1,121 @@
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="name", type="string", nullable=true, length=200)
```
not nullでuniqueかと思います。

sai-gillingham replied:
修正しました

kiy0taka commented on Entity/TwoFactorAuthType.php:
```diff
@@ -0,0 +1,121 @@
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="route", type="string", nullable=true, length=200)
```
not nullでuniqueかと思います。

sai-gillingham replied:
修正しました

kiy0taka commented on EventListener/CustomerTwoFactorAuthListener.php:
```diff
@@ -0,0 +1,411 @@
+    private function isActivationRoute(string $route): bool
+    {
+        // ルートで認証
+        if (in_array($route, [self::ACTIVATE_ROUTE])) {
```
`$route === self::ACTIVATE_ROUTE`

sai-gillingham replied:
こちらの関数をCustomerPersonalValidationListenerクラスに移動しましたが、
修正しました。

kiy0taka commented on EventListener/CustomerTwoFactorAuthListener.php:
```diff
@@ -0,0 +1,411 @@
+    }
+
+    /**
+     * リクエスト受診時イベントハンドラ.
```
/受診/受信/

sai-gillingham replied:
修正しました。

kiy0taka commented on EventListener/CustomerTwoFactorAuthListener.php:
```diff
@@ -0,0 +1,411 @@
+     * @param mixed $event
+     * @throws HttpException\NotFoundHttpException 
+     */
+    public function deviceAuth($event) 
```
`private` で良いと思います。

sai-gillingham replied:
こちらの関数を`CustomerPersonalValidationListener`クラスに移動しましたが、
`deviceAuth`のアクセス修飾子を`private`にしました。

kiy0taka commented on EventListener/CustomerTwoFactorAuthListener.php:
```diff
@@ -0,0 +1,411 @@
+     * @throws HttpException\NotFoundHttpException 
+     * @return mixed
+     */
+    public function multifactorAuth($event, $Customer, $route, $uri) 
```
`private` で良いと思います。

sai-gillingham replied:
修正しました。

kiy0taka commented on Entity/TwoFactorAuthConfig.php:
```diff
@@ -0,0 +1,274 @@
+     *
+     * @ORM\Column(name="from_tel", type="string", nullable=true, length=200)
+     */
+    private $from_tel = null;
```
`$from_phone_number`

sai-gillingham replied:
修正しました。

kiy0taka replied:
`getFromPhoneNumber` / `setFromPhoneNumber` でお願いします。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/blob/9e48ad2dbb15ec9ccb2cf3a7dffbe46427f0fba8/Entity/TwoFactorAuthConfig.php#L139

sai-gillingham replied:
修正しました。

kiy0taka commented on Event.php:
```diff
@@ -0,0 +1,126 @@
+    }
+
+    /**
+     * [/admin/setting/shop]表示の時のEvent Fork.
```
/Event Fork/Event Hook/g

sai-gillingham replied:
修正しました。

kiy0taka commented on Repository/TwoFactorAuthConfigRepository.php:
```diff
@@ -0,0 +1,56 @@
+     */
+    public function findOne()
+    {
+        $qb = $this->createQueryBuilder('c')->setMaxResults(1)->select('c');
```
`return $this->findOneBy([])` で良いと思います。

sai-gillingham replied:
修正しました。

kiy0taka commented on Service/CustomerTwoFactorAuthService.php:
```diff
@@ -0,0 +1,345 @@
+    /**
+     * @var int デフォルトの認証の有効日数
+     */
+    public const DEFAULT_EXPIRE_DATE = 14;
```
パラメータの設定はAPIプラグインのようにservices.yamlでデフォルト値や環境変数から設定できるようにお願いします。
https://github.com/EC-CUBE/eccube-api4/blob/4.2/Resource/config/services.yaml#L4-L14

sai-gillingham replied:
修正しました。
上記の定数の値は
plugin_eccube_2fa_customer_expire設定変数に載せました。

kiy0taka commented on Service/CustomerTwoFactorAuthService.php:
```diff
@@ -0,0 +1,345 @@
+     *
+     * @return boolean
+     */
+    public function isAuth($Customer, $route = null)
```
`isAuthed`

sai-gillingham replied:
修正しました。

kentanakayrglm committed:
・PR対応（不要な処理・変数・useの削除）
・管理画面のバリデーション不具合修正
・スロットリングの設定追加

kentanakayrglm committed:
- デフォルトログイン認証ルートの設定不具合修正

sai-gillingham committed:
Merge pull request #1 from KenTanaka/main

修正後のバージョンを取得する

sai-gillingham committed:
デバイス認証済み電話番号をnullにしたら、ECCUBEシステムがバグする。

sai-gillingham committed:
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/5
ログアウト時に2段階認証画面にアクセスするとシステムエラー
の修正。

sai-gillingham committed:
ゲスト登録不具合の修正。

sai-gillingham committed:
本人認証が無効設定でも実行されるの修正
本人認証と二段階認証のイベントリスナーを分割すること。

sai-gillingham committed:
本人認証が無効設定でも実行されるの修正
本人認証と二段階認証のイベントリスナーを分割すること。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/7

sai-gillingham committed:
Merge remote-tracking branch 'origin/main'

sai-gillingham committed:
会員編集画面の2段階認証方式項目に未設定オプションを追加する。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/11

sai-gillingham committed:
仮会員設定：無効
本人認証：無効
二段階認証：有効
パターンの対応。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/7

sai-gillingham committed:
購入フローで二段階認証をスキップさせないように修正。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/6

sai-gillingham committed:
APP認証プラグインを無効化した状態で、二段階認証にAPPが表示されないように修正
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/12

sai-gillingham committed:
無効になっているTwoFactorAuthTypeの行を消すことより無効になっているTwoFactorAuthType行を無効に設定する
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/12

kentanakayrglm committed:
マイページ配下のページを重要操作ルーティングに指定しても、再認証されていない不具合を修正
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/14

sai-gillingham committed:
２段階認証選択画面の選択肢修正。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/12

sai-gillingham committed:
Merge remote-tracking branch 'origin/main'

sai-gillingham committed:
２段階認証を利用する項目は不要になったため、two_factor_auth項目を消す。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/12

sai-gillingham committed:
２段階認証を利用する項目は不要になったため、two_factor_auth項目を消す。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/12

sai-gillingham committed:
ワンタイムパスワードの値をハッシュ化にする。+ SMSの電話番号チェックの修正
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/13

sai-gillingham committed:
ログアウトするときに、二段階認証クッキーが消えるようになった。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/15

sai-gillingham committed:
デフォルト操作ルートのみで２段階認証クッキーデータをデータベースに保存する
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/17

sai-gillingham committed:
ハッシュしているため、device_auth_one_time_token文字列のlengthを10桁から255桁に変更する。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/13

sai-gillingham committed:
e2e テスト。

sai-gillingham committed:
e2e テスト。

sai-gillingham committed:
e2e テスト。

sai-gillingham committed:
一旦、手動のみで実行する。

sai-gillingham committed:
一旦、手動のみで実行する。

sai-gillingham committed:
add env=dev.

sai-gillingham committed:
env fix.

sai-gillingham committed:
AB base test file.

sai-gillingham committed:
E2E Include all plugins.

sai-gillingham committed:
AB TEST.

sai-gillingham committed:
AB TEST.

sai-gillingham committed:
AB TEST.

sai-gillingham committed:
mounted mode.

sai-gillingham committed:
repository name fix.

sai-gillingham committed:
clean all files.

sai-gillingham committed:
DEBUG.

sai-gillingham committed:
MYSQL debug.

sai-gillingham committed:
DOCKER ISSUE?

sai-gillingham committed:
MYSQL load issue.

sai-gillingham committed:
MYSQL load issue.

sai-gillingham committed:
FIXUP WORDING.

sai-gillingham committed:
Permissions issue?

sai-gillingham committed:
除外ルーティング項目を消す。

chihiro-adachi committed:
composer.json更新

sai-gillingham committed:
デフォルト操作ルート
＋
重要操作ルート
で２段階認証クッキーデータをデータベースに保存する

sai-gillingham committed:
gitflowの順番変更。

sai-gillingham committed:
gitflowの順番変更。

sai-gillingham committed:
PluginManagerから除外ルーティング設定処理を消す

sai-gillingham committed:
本人認証と二要素認証クラスの別け

sai-gillingham committed:
\Twig_Environmentを\Twig\Environmentに変更。

sai-gillingham committed:
性能試験：実行順番変更

sai-gillingham committed:
性能試験：実行順番変更

sai-gillingham committed:
TokenStorageInterfaceは不要のため、消す

sai-gillingham committed:
フォーム項目の値の抽出をisValid|isSubmittedの範囲内に移動する。

sai-gillingham committed:
性能試験：フアイル権限の変更

sai-gillingham committed:
findBy -> findOneByに移動

sai-gillingham committed:
不要変数を消す

sai-gillingham committed:
RedirectResponseを$this->redirectToRouteに変更する。

sai-gillingham committed:
性能試験：sudoで過去のフアイルを消す

chihiro-adachi committed:
Merge pull request #21 from EC-CUBE/fix-composer-json

composer.json更新

sai-gillingham committed:
composer json の更新

sai-gillingham committed:
リポジトリクエリーをfindOneBY関数に変更

sai-gillingham committed:
array_sliceの不具合

sai-gillingham committed:
plg_two_factor_auth_type.name列をNULLからNOT NULLに変更する。

sai-gillingham committed:
他の会員でも同じ番号によって認証済みになってないかの確認コードの追加。

sai-gillingham committed:
不要変数を消す

sai-gillingham committed:
字の誤りの修正

sai-gillingham committed:
不要定数を消す。

kentanakayrglm committed:
本人認証：トークン入力時の電話番号確認処理 考慮漏れを修正

kentanakayrglm committed:
表記ゆれ修正

kentanakayrglm committed:
概要資料更新

kentanakayrglm committed:
本人認証エンドポイントのチェック修正（認証済みか？→ログイン済みか？）

kentanakayrglm committed:
本人認証完了時、認証トークンをクリアする

kentanakayrglm committed:
不要コメント削除

kentanakayrglm committed:
フィールド属性を修正

chihiro-adachi committed:
Merge pull request #22 from sai-gillingham/main

レビュー指摘修正版

chihiro-adachi committed:
fix cs

kentanakayrglm committed:
・composer - version修正
・twig/メッセージリソースの調整

kentanakayrglm committed:
マージ漏れ修正
　- 不要ページの登録削除
    - twig名の修正

sai-gillingham committed:
ページデータが消えない問題の修正
＋
残り除外ルーティング処理の削除
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/24

sai-gillingham committed:
ページデータが消えない問題の修正
＋
残り除外ルーティング処理の削除

sai-gillingham committed:
PluginManagerの分岐条件の更新

sai-gillingham committed:
PluginManagerの分岐条件の更新
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/24

sai-gillingham committed:
Merge remote-tracking branch 'origin/main'

chihiro-adachi committed:
secret_keyのバリデーションを追加

chihiro-adachi committed:
$Customerのnullチェックを追加

chihiro-adachi committed:
未使用の関数を削除

chihiro-adachi committed:
厳密にnullチェック

chihiro-adachi committed:
コメント微調整

KenTanaka committed:
Merge pull request #23 from EC-CUBE/fix-cs

確認しました。

KenTanaka committed:
Merge pull request #26 from EC-CUBE/fix-review

Controller/CustomerPersonalValidationController.php

kentanakayrglm committed:
バージョン修正

sai-gillingham committed:
競合の修正

KenTanaka committed:
Merge pull request #28 from sai-gillingham/main

・レイアウト調整
・PluginManager不具合修正

sai-gillingham committed:
CS FIXER + PhpStormクリーンアップ機能

sai-gillingham committed:
不要変数を消す

sai-gillingham committed:
不要変数を消す

sai-gillingham committed:
不要変数を消す+CS FIXER

sai-gillingham committed:
変数回復

sai-gillingham committed:
Twigファイルのコメントの修正

sai-gillingham committed:
Private/public順番の修正

sai-gillingham committed:
$createdAt|$updatedAtアクセス修飾子の修正

kurozumi committed:
sendBySmsメソッドの不要な引数を削除

TwoFactorAuthCustomerSms42では$Customerは引数に指定されていませんし、
メソッド内で$Customerは使用されてないのでsendBySmsメソッドの引数から削除しました。

あとsendBySmsメソッドは汎用性がなさそうなのでTwoFactorAuthCustomerSms42に組み込んだほうが良いんじゃないかと思いました。

kentanakayrglm committed:
ログアウト時、ログアウト会員の二段階認証クッキーレコードを削除する処理を追加 https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/15

sai-gillingham committed:
コードの整理

sai-gillingham committed:
改行の追加

KenTanaka committed:
Merge pull request #29 from sai-gillingham/main

dotani1111 committed:
fix:項目名の変更

chihiro-adachi committed:
Merge pull request #34 from dotani1111/features/fix_two_factor_auth_name

fix:「二段階認証」の一部項目名を変更

chihiro-adachi committed:
Merge branch 'main' into kurozumi-patch-1

chihiro-adachi committed:
Merge pull request #31 from kurozumi/kurozumi-patch-1

sendBySmsメソッドの不要な引数を削除

chihiro-adachi committed:
マイページ配下に変更

chihiro-adachi committed:
ツールチップを表示するように修正

sai-gillingham committed:
二段階認証のプラグイン（APPorSMS）がない状態であれば、二段階認証項目の非表示状態になる
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/33

sai-gillingham committed:
デバッグ用のdisabled属性を消す
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/33

sai-gillingham committed:
Merge pull request #3 from sai-gillingham/review/pg-review

PGレビュー

KenTanaka committed:
Merge pull request #37 from EC-CUBE/fix-tooltip

確認しました。

sai-gillingham committed:
APIトークン項目のマスキング
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/19

sai-gillingham committed:
APIトークン項目のマスキング
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/19

sai-gillingham committed:
不要変数を消す
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/33

sai-gillingham committed:
ワンタイムパスワード・マジックナンバーの生成処理のコード整理
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/35

sai-gillingham committed:
ワンタイムパスワードの桁数を環境変数から定数に移動する。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/issues/35

dotani1111 committed:
Merge pull request #38 from sai-gillingham/hotfix/hide_2_auth_option_on_disabled

２段階認証項目表示の変更

dotani1111 committed:
Merge pull request #39 from sai-gillingham/feature/hide_api_token_v2

APIトークン項目マスキングの実施

chihiro-adachi committed:
設計書をdoc配下へ移動

chihiro-adachi committed:
リリース用のactionsを追加

chihiro-adachi committed:
v2に更新

sai-gillingham committed:
Merge branch 'EC-CUBE:main' into main

KenTanaka committed:
Merge pull request #40 from chihiro-adachi/add-release-action

Add release action

KenTanaka committed:
Merge pull request #36 from EC-CUBE/fix-routing

ルーティングをマイページ配下に変更

sai-gillingham committed:
Merge branch 'EC-CUBE:main' into main

KenTanaka committed:
Merge pull request #41 from sai-gillingham/feature/organize_one_time_code

ワンタイムパスワード・マジックナンバーの生成処理のコード整理

sai-gillingham committed:
Merge branch 'EC-CUBE:main' into main

kiy0taka commented:
こちらはルーティング名と合わせておいてください。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/blob/9e48ad2dbb15ec9ccb2cf3a7dffbe46427f0fba8/Resource/config/services.yaml#L3

sai-gillingham committed:
findOneByではなくcountを利用するように修正しました。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#discussion_r1147157803

sai-gillingham committed:
ワンタイムパスワードの桁数を定数から環境変数に移動する。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#discussion_r1147164107

sai-gillingham committed:
ゲッター・セッター関数名のリファクタリング
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#discussion_r1147172574

sai-gillingham committed:
スロットリングパラメータはルーティング名と合わせました。
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/3#issuecomment-1482332934

sai-gillingham committed:
SMSトークンの長さのオーバーライドパラメータとSMS有効期限のオーバーライドパラメータを消す
https://github.com/EC-CUBE/TwoFactorAuthCustomer42/pull/42

sai-gillingham commented:
> こちらはルーティング名と合わせておいてください。
> 
> https://github.com/EC-CUBE/TwoFactorAuthCustomer42/blob/9e48ad2dbb15ec9ccb2cf3a7dffbe46427f0fba8/Resource/config/services.yaml#L3

修正しました。

kiy0taka committed:
Merge pull request #42 from sai-gillingham/feature/review_fixes_2023_03_24_1

レビュー用PR_2023_03_24に関しての修正
